src/client/pages/OfferNew/Introduction/Sidebar/DetailsModal/Details/components/index.tsx

```diff
@@ -0,0 +1,3 @@
+export * from './styledComponents'
```

Comment:
Yes, I was going back and forth between those names! The thing about styles is that it might be confused with only styling objects, but since you only use emotions that is maybe not the case. I will change!

Reply:
Should we really use index files for exports? I would argue for just using named exports and get rid of extra files 🙂

Reply:
Also, I agree with that we should use`.styles` but also some more context to the filename. Like in this case it's styling for the Details components so what do you think about `Details.styles.tsx` ?

Reply:
I like the Details.styles.tsx suggestions because it gets easier to navigate the project, if  you search the project for Details. Then you see the Details.styles file as well.